spec/assigned_test_spec.rb

```diff
@@ -16,6 +16,25 @@ module Ab
     let(:buckets) { 1..1000 }
     let(:thousand_variants) { 1.upto(1000).map { |i| AssignedTest.new(test, i).variant } }
 
+    context 'with non existent variant' do
+      let(:variants) { [OpenStruct.new(name: 'enabled', accumulated_chance_weight: 2)] }
+
+      subject { assigned_test.disabled? }
+      it { should be false }
```

Comment:
Currently the gem uses should syntax:
```
RSpec.configure do |config|
  config.expect_with(:rspec) { |c| c.syntax = :should }
  config.mock_with(:rspec) { |c| c.syntax = :should }
end
```

Could be changed with separate PR though